packages/compose/README.md

```diff
@@ -157,6 +157,29 @@ _Returns_
 
 -   `boolean`: Reduced motion preference value.
 
+<a name="useResizeObserver" href="#useResizeObserver">#</a> **useResizeObserver**
+
+Hook which allows to listen the resize event of any target element when it changes sizes.
+
+_Usage_
+
+```js
+const App = () => {
+	const [ resizeListener, sizes ] = useResizeObserver();
+
+	return (
+		<div>
+			{ resizeListener }
+			Your content here
+		</div>
+	);
+};
+```
+
+_Returns_
+
+-   `Object`: Measurements object with properties `width` and `height`.
```

Comment:
I think we should be very clear about how the value will be `null` on the first render, since otherwise it seems it could be a very common source of errors.

Reply:
Will mention about it!